website/docs/getting-started/for-designers.md

```diff
@@ -118,9 +118,9 @@ Open the [resources menu](https://help.figma.com/hc/en-us/articles/360039150413-
 
 #### Copying from the stickersheet
 
-If you prefer to select a component visually, the [Product Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?t=Ooe3pkDap3cGcgAH-1) library has a stickersheet for each component displaying the available variants and properties. Just like when using the direct inserts, this method maintains the link to the main component in the library, which means it will receive updates as they become available.
+If you prefer to select a component visually, the [Product Components](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67397-95918&t=gWdKy44MzTP4cTRo-1) library has a stickersheet for each component displaying the available variants and properties. Just like when using the direct inserts, this method maintains the link to the main component in the library, which means it will receive updates as they become available.
 
-1. Open the [Product Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?t=Ooe3pkDap3cGcgAH-1) library.
+1. Open the [Product Components](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&t=zQtMoxS3oB4AplFG-6) library.
```

Comment:
This link also leads to the Button stickersheet

Reply:
This one probably makes more sense to link the README 👍